### DISCUSSION TITLE
Discourse-server seems dead

### DISCUSSION THREAD

**Discussion by NcMichel** [ImageMagick/ImageMagick#2578] [Help]
Hi
https://www.imagemagick.org/discourse-server/

seems to be dead 
**_Not Found
The requested URL /discourse-server/ was not found on this server._**

Its a big loss for beginners like me since a lot of simple answers were there, and a big loss for community since at the end of the day we ask the same questions again....
Any way to restore it ?

Thanks 
Michel

**Comment by dlemstra** [comment]
The spotlighted discussion should explain why: https://github.com/ImageMagick/ImageMagick/discussions/2565

**Comment by NcMichel** [comment]
Does that mean we can't access until some weeks, or is it in archive ?

Le mar. 15 sept. 2020 à 18:53, Dirk Lemstra <notifications@github.com> a
écrit :

**Reply by fmw42** [reply]
https://archive.imagemagick.org/discourse-server/
